libparsec/crates/types/src/fixtures.rs

```diff
@@ -1,5 +1,7 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
+#![allow(clippy::unwrap_used)]
+
```

Comment:
I would play dumb => because we use `unwrap` in that module.

Maybe you want a comment why it's okay to have `unwrap` for the module `fixture` and not somewhere else ?

Reply:
Exact. I am not a Rust expert so I might be missing something here. My reasoning is this: We want to limit `unwrap` usage to the tests. This file uses `unwrap` but is not in the test folder so we are adding an exception. Is this temporary until we remove the `unwrap`s? is it ok because the file is only used for testing purposes? other reason?

Reply:
My first reasoning is that module is used for testing purpose so I want the same rules / exceptions as for the tests code

Reply:
So if it is not a temporary allow, then the comment is not that necessary :+1: